docs/guides/cli/05_cli-basics.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: cli-basics
+title: Basic functionality
```

Comment:
Maybe

```suggestion
title: First Steps with Ory CLI 
```

Reply:
I see you like the "First steps" phrase, but let's not apply "first steps" treatment to everything :D I'd stick to my title and maybe update it further along the way when we re-organize the docs further.

Reply:
Ok, then make the F upper case to follow our NYT style for headlines :) 

```suggestion
title: Basic Functionality
```

We can change that rule but we need to change it everywhere if we do for consistency